vllm/model_executor/layers/fused_moe/fused_moe.py

```diff
@@ -1406,9 +1425,14 @@ def fused_moe(
 
     if use_grouped_topk:
         assert num_expert_group is not None and topk_group is not None
-        topk_weights, topk_ids = grouped_topk(hidden_states, gating_output,
-                                              topk, renormalize,
-                                              num_expert_group, topk_group)
+        topk_weights, topk_ids = grouped_topk(
+            hidden_states,
+            gating_output,
+            topk,
+            renormalize,
+            num_expert_group,
+            topk_group,
+            share_fusion=envs.VLLM_SHARED_EXPERT_FUSION_REPLICAS)
```

Comment:
Generally the argument name `share_fusion` sounds like it's a bool. Could you rename it to something like `num_share_fusion_replicas`?